add-translation-keys

From: Bryan Larsen <bryan@larsen.st>

To add translations, you must add translation keys to your code.

For example, to translate one of the strings on the front page:

SHOW_PATCH
---
 app/views/front/index.dryml |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/app/views/front/index.dryml b/app/views/front/index.dryml
index 8af97d4..936e9af 100644
--- a/app/views/front/index.dryml
+++ b/app/views/front/index.dryml
@@ -24,7 +24,7 @@
     </header>
 
     <section with="&current_user" class="content-body" if="&logged_in?">
-      <h3>Your Projects</h3>
+      <h3><t key="front_title">Your Projects</t></h3>
       <collection:projects><card without-creator-link/></collection>
 
       <a:projects action="new">New Project</a>
